wom/src/main/scala/wom/expression/WomExpression.scala

```diff
@@ -128,6 +128,13 @@ trait IoFunctionSet {
   // Functions that do NOT necessitate network I/O but are only manipulating paths
   def pathFunctions: PathFunctionSet
 
+  /**
+    * Get the basename of this path. If a DRS path, resolve to a real URL and get the basename
```

Comment:
nice docs!

Reply:
In this vein should we add a note on `name()` above that it might not be the function one is looking for?

Reply:
Done!